grid-cli/cmd/deploy_vm.go

```diff
@@ -143,7 +153,7 @@ var deployVMCmd = &cobra.Command{
 		}
 
 		if node == 0 {
-			filter, disks, rootfss := filters.BuildVMFilter(vm, mount, farm)
+			filter, disks, rootfss := filters.BuildVMFilter(vm, diskMount, farm)
```

Comment:
the volume mount should also be accounted for in building the filter

Reply:
does volume will be passed to filterNodes( )? , to know if i have to return it from BuildVmFilter or not